src/coreclr/tools/Common/Compiler/Logging/ReferenceSource/CompilerGeneratedState.cs

```diff
@@ -1,6 +1,5 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
```

Comment:
Files under ReferenceSource are literal copies from the dotnet/linker repo. It doesn't make much sense updating them here. The change will be stomped over next time we do integration.

Reply:
Yeah, I've rolled it back.

Reply:
Updated license upstream as well https://github.com/dotnet/linker/pull/2303.